<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplayTabWidget</class>
 <widget class="QWidget" name="ReplayTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReplayTabWidget</string>
  </property>
  <layout class="QVBoxLayout" name="layoutReplayTabWidget">
   <item>
    <widget class="QGroupBox" name="playbackBox">
     <property name="title">
      <string>Playback</string>
     </property>
     <layout class="QVBoxLayout" name="layoutPlayback">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="text">
         <string>Title: None</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="playbackProgress">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTime">
        <property name="text">
         <string>00:00/00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutButtons">
        <item>
         <widget class="QPushButton" name="buttonReplay">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Replay</string>
          </property>
          <property name="icon">
           <iconset resource="InputRepeater.qrc">
            <normaloff>:/Resources/PlayIcon.png</normaloff>:/Resources/PlayIcon.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonStop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
          <property name="icon">
           <iconset resource="InputRepeater.qrc">
            <normaloff>:/Resources/StopIcon.png</normaloff>:/Resources/StopIcon.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelWarning">
          <property name="text">
           <string>&lt;font color=&quot;red&quot;&gt;Warning! Stopping replay before finish may leave some keys in &lt;b&gt;pressed&lt;/b&gt; state without emitting &lt;b&gt;released&lt;/b&gt; signal</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="optionBox">
        <property name="title">
         <string>Playback options</string>
        </property>
        <layout class="QVBoxLayout" name="layoutOptionBox">
         <item>
          <layout class="QHBoxLayout" name="layoutSpeed">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLabel" name="labelSpeed">
             <property name="text">
              <string>Speed:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinSpeed">
             <property name="decimals">
              <number>1</number>
             </property>
             <property name="minimum">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layoutRepeat">
           <item>
            <widget class="QLabel" name="labelRepeatCount">
             <property name="text">
              <string>Repeat count:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinRepeatCount">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000000000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkRepeatInfinitely">
           <property name="text">
            <string>Repeat infinitely</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="InputRepeater.qrc"/>
 </resources>
 <connections/>
</ui>
